src/main/resources/jupyter/safe-mode.js

```diff
@@ -0,0 +1,105 @@
+// Adapted from the All of Us Researcher Workbench "Playground Mode"
+// https://github.com/all-of-us/workbench/blob/master/api/cluster-resources/playground-extension.js
+
+// TODO: the following is required until IA-979 is implemented
+// To deploy on a Leonardo cluster:
+// 1. copy to a public GCS location
+// 2. issue a PUT request to <leonardo>/api/cluster/<billing project>/<cluster name>
```

Comment:
Interesting - does this get picked up immediately? Or does the cluster need to restart?

Reply:
This is the cluster create endpoint.  Any new notebooks on the cluster have the extension enabled.